### PR TITLE
Metadata checker

### PR DESCRIPTION
**Pull request by adamrp**
Adds an additional check to see if metadata can be retrieved for a particular sample.  Note that this touches python code, so the web server will have to be restarted after deployment.

Good barcodes you might use to test:
000001002 (should be all good)
000005739 (this is an animal sample, so metadata pulldown fails)
010001002 (this sample does not exist)
000001055 (this sample is missing sample data)

### CODE REVIEW

**Comment by adamrp** [comment]
@ElDeveloper and @teravest, can this get tested ASAP?  Now that geocoding of all samples has finished, it would be awesome to get this merged so that we can move forward with AG round 7 (those plating the samples will be able to use this to ensure that nothing is sequenced for which we can't successfully pull down the metadata, which has been a problem in the past that has theoretically been fixed now).

**Comment by teravest** [comment]
I will test it today.

**Comment by adamrp** [comment]
Cool, thanks!

**Comment by adamrp** [comment]
I think I have addressed all the comments so far, so unless there are more, this should be ready to go!

**Comment by teravest** [comment]
Looks good to me

File: www/wwwadmin/barcode_util.psp [hunk 1]
```diff
@@ -137,9 +135,20 @@ try:
...
+                email_type = "1"
+            elif len(barcode_metadata) == 0:
+                # we cannot retrieve metadata
+                div_id = "no_metadata"
+                message = "Cannot retrieve metadata"
+            else:
+                # should never get here (this would happen if the metadata
+                # pulldown returned more than one row for a single barcode)
+                div_id = "md_pulldown_error"
+                message = "Please notify Adam that you saw this message..."
```
Comment (ElDeveloper) [inline www/wwwadmin/barcode_util.psp]: Could you change this message to describe the problem and add suggest contacting one of the DB admins.

**Comment by ElDeveloper** [comment]
From what we discussed during our meeting today and to have a record of this:
- [ ] Remove the query that does this on bulk for a set of barcodes. This includes removing this query from the repo, `git rm` is your friend.
- [ ] Add interface on python to replace the other query.
- [ ] Make a simple script under `python_code/scritps/`.

**Comment by adamrp** [comment]
Okay, how does this look?  Tested a little bit, seems to work fine.

**Comment by ElDeveloper** [comment]
@adamrp Would you mind moving `ag_get_metadata.py` to live under `python_code/scripts/`?

**Comment by ElDeveloper** [comment]
This looks just like what we want.

**Comment by adamrp** [comment]
Will do!

**Comment by ElDeveloper** [comment]
Cool, thanks!

**Comment by adamrp** [comment]
Okay, this should be ready for merge (perhaps after a re-review of the new changes)

**Comment by ElDeveloper** [comment]
Re-reviewing right now, thanks for making the changes!

**Comment by ElDeveloper** [comment]
:+1:

**Pull request merged**